README.md

```diff
@@ -59,6 +59,8 @@ important fields are:
   run.
 - `upload_bundle` A boolean value. When set to true, it uploads the bundle to the Play Store,
   Otherwise, it uploads the APK.
+- `kiwix_organization` A boolean value. When set to true, it uploads the application in kiwix organization,
```

Comment:
Why you need this, you just need to check if there is a dedicated secret with the app tag, and otherwerise fallback to the kiwix json?!

Reply:
Thanks for your suggestion.